src/style.css

```
@@ -1,3 +1,82 @@
-body {
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body,
+ul,
+li {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  align-items: center;
+  justify-content: space-between;
+  background: #fff;
+}
+
+.logo__img {
+  height: 40px;
+  width: 40px;
+  display: block;
+}
+
+.nav {
+  display: inline-block;
+  height: 100%;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
   margin: 0;
+  width: 100%;
+  list-style: none;
+  height: 100%;
+}
+
+.nav__link {
+  display: flex;
+  text-decoration: none;
+  justify-content: center;
+  align-items: center;
+  font-size: 12px;
+  text-transform: uppercase;
+  color: #000;
+  font-style: normal;
+  font-weight: 500;
+  height: 100%;
+  padding: 23px 0;
+}
+
+.nav__item {
+  margin-right: 20px;
+  height: 100%;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.is-active::after {
+  content: "";
+  display: block;
+  height: 4px;
+  background: #00acdc;
+  border-radius: 8px;
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+}
+
+.is-active,
+.nav__link:hover {
+  color: #00acdc;
 }
```

```suggestion
.nav__link:hover {
  color: #00acdc;
}
```
redundant for .is-active. already settled above